spec/request/sidekiq_admin_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Sidekiq admin', type: :request do
+  let(:user) { create(:user) }
+  let(:admin_user) { build_stubbed :user,
+                                   given_name: 'Joe',
+                                   surname: 'Developer',
+                                   groups: [Group.new(name: 'cn=umg/up.dlt.scholarsphere-admin,dc=psu,dc=edu')] }
```

Comment:
@whereismyjetpack how about a trait here. `let(:user) { build(:user, :is_admin) }` Then you define a trait in `spec/factories/users.rb` that adds the admin group. The given and surname are inconsequential and get generated by the factory.

Reply:
Thanks for that, I wasn't sure how best to handle that. I knew pasting that umg everhwere smelled bad, just couldn't tell where the smell was coming from.